### DISCUSSION TITLE
Can’t get past pre-check on student discount pack application

### DISCUSSION THREAD

**Discussion by avvy695** [community/community#42240] [GitHub Education]
I’m struggling to submit an application for the student discount pack, as I can’t get past the pre-check. Each time I get the following message:

_Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.,You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so._

I have completed all the fields on my GitHub user profile, and although I am currently away from campus I don’t think this should be stopping me from actually submitting the application. Any ideas on how to actually get it submitted?

**Comment by xixilaladsds** [comment]
The same with you.  I just leave my message and wait for official's apply.

**Comment by nanashi-w** [comment]
I'm in the same boat, I've followed the official tutorial and simply set up my profile, but it still shows this prompt
